server/config/options.go

```diff
@@ -73,6 +73,7 @@ var DefaultConfig = Config{
 		Port:         8108,
 		ReadEnabled:  true,
 		WriteEnabled: true,
+		AuthKey:      "ts_test_key",
```

Comment:
it is needed if the server is not running inside docker.

Reply:
Instead of doing this you can just run `make local_run`

Reply:
is that setting environment variables locally?

Reply:
It uses `config/server.dev.yaml` to start the binary:
```
local_run: server
	$(DOCKER_COMPOSE) up --no-build --detach tigris_search tigris_fdb
	TIGRIS_ENVIRONMENT=dev ./server/service
```
After `make local_run` you can even attach to and debug the server from the IDE.

Reply:
but then I need to attach to remote process? I won't be able to simply run from the IDE, is that right?

Reply:
You can run from IDE if you set `TIGRIS_ENVIRONMENT=dev` in the IDE.

Reply:
When you attach, it's local process on you host.
You can set breakpoints and debug as usual, not much different from running from the IDE.

Reply:
So TIGRIS_ENVIRONMENT=dev still making sense to me but otherwise why introduce one more thing for the developer? All we need is to load the config object from the yaml based on the env.

Reply:
I have removed it but curl is not working for me when I am running the server using `make local_run`. Is it working for you?

Reply:
It should. Let's check tomorrow, why it's not.